docs/technical_reference/unit_models/Biploar_electrodialysis_0D.rst

```diff
@@ -0,0 +1,340 @@
+Bipolar Electrodialysis (0D)
+============================
+
+Introduction
+------------
+
+Bipolar electrodialysis, an electrochemical separation technology, has primarily been used to generate acids and bases
+from waste salts produced in water purification. By providing in-situ access to valuable raw materials bipolar membranes can
+significantly reduce the total cost of the operation.  This cell stack is shown in Figure 1 with **basate** and **acidate** channels, that produce base and acid
+respectively. More overview of the bipolar
+electrodialysis technology can be found in the *References*.
+
+.. figure:: ../../_static/unit_models/BPEDdiagram.png
+    :width: 600
+    :align: center
+
+    Figure 1. Schematic representation of a bipolar electrodialysis unit
+
+
+One bipolar membrane along with the **Acidate** and **Basate** channels can thus be treated as a modelling unit that can
```

Comment:
Are acidate and basate typical terminology for BPMD?

Reply:
They are not. I needed a name for the acid & base channel. 
I went off diluted/ concentrated channel which are called diluate/concentrate (in ED documentation). Extrapolating from this I chose acidate/basate.

If anybody has better suggestion I am happy to use it.

Reply:
In theory they are an acid concentrate and base concentrate (increasing in concentration), where the brine is a brine diluate (deceasing in concentration) based on the donation of ions toward the objective of the system, correct?

Reply:
Yes, they do increase concentration. But I think there is a subtle difference. BPMED increase concentration via production of new acid/base (through water splitting) and not by just transporting ions across the membranes. I feel like that deserved a distinction in nomenclature. Thoughts?

Reply:
I'd go with either acid and base or acidic and basic.

Reply:
@adam-a-a That's reasonable. I have changed the names to acidic & Basic